pkgs/desktops/xfce/applications/xfmpc/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, autoreconfHook
+, xfce4-dev-tools
+, libxfce4ui
+, libxfce4util
+, pkg-config
+, vala
+, gtk3
+, glib
+, libmpd
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xfmpc";
+  # Last release is too old
+  version = "0.3.1-2024-05-29";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.xfce.org";
+    owner = "apps";
+    repo = "xfmpc";
+    rev = "cf40dffec6e9b80abb1f1aa6d7dceef4790173dc";
+    hash = "sha256-moCWSLGBJuWM4/lRJi6D3w38iJeCntLo3Vl/eVfu7lw=";
+  };
+  nativeBuildInputs = [
+    autoreconfHook
+    xfce4-dev-tools
+    pkg-config
+    vala
+    # Needed both here and in buildInputs for cross compilation to work
+    libxfce4ui
+    libxfce4util
+  ];
+  buildInputs = [
+    gtk3
+    glib
+    libxfce4ui
+    libmpd
+  ];
+  configureFlags = [
+    # From some reason, upstream conditioned this maintainer mode with the
+    # search of the vala compiler in `configure.ac`. Without `VALAC` set in the
+    # makefiles, the build fails with a wierd and hard to understand error.
+    "--enable-maintainer-mode"
```

Comment:
`mkXfceDerivation` is always passing this:

https://github.com/NixOS/nixpkgs/blob/b57526ee5cf47504b759cda0b3655a73b40e9a61/pkgs/desktops/xfce/mkXfceDerivation.nix#L30

[Release tarball](https://archive.xfce.org/src/apps/xfmpc/) dists generated C files so I think neither the maintainer mode nor valac will be needed there. It looks like devs  [don't quite understand](https://matrix.to/#/!lhGQfSoqXljRjKiRMt:matrix.org/$7cHH2oMaUcMnkNYZ8Zc5cH8gwyR2_0OWxDQxBsysjTM?via=matrix.org&via=tarricone.org&via=tchncs.de) why we are not using release tarballs 🙃

Reply:
> It looks like devs [don't quite understand](https://matrix.to/#/!lhGQfSoqXljRjKiRMt:matrix.org/$7cHH2oMaUcMnkNYZ8Zc5cH8gwyR2_0OWxDQxBsysjTM?via=matrix.org&via=tarricone.org&via=tchncs.de) why we are not using release tarballs 🙃

Couldn't open that link.. But I understand it's not super important to understand it.

Reply:
> `mkXfceDerivation` is always passing this:

In general it's a good idea to use it!